<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCParameterWidget</class>
 <widget class="QFrame" name="CCParameterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#CCParameterWidget{
	background:transparent;
}

QLabel{
	font: 12pt &quot;微软雅黑&quot;;
	color:white;
	background: transparent;
}

QLineEdit {   
	border: 1px solid white;
	selection-background-color: #0A246A;
	font: 12pt &quot;微软雅黑&quot;;
	color:white;
	background: transparent;
}   

QLineEdit[echoMode=&quot;3&quot;] {   
    lineedit-password-character: 9679;   
}
   
QLineEdit:read-only { 
	color: darkgray;
    background:transparent;   
}

QSpinBox, QDoubleSpinBox, QDateTimeEdit {
	border: 1px solid white;
	font: 12pt &quot;微软雅黑&quot;;
	color:white;
	background: transparent;
}

QSpinBox::up-button, QDoubleSpinBox::up-button, QDateTimeEdit::up-button {
	subcontrol-origin:border;
	subcontrol-position:right;
	padding-left: 10px;
	image: url(:/cellculture_icons/arrow_right.png);
	width: 24px;height: 28px; right:5px;
}
QSpinBox::down-button, QDoubleSpinBox::down-button, QDateTimeEdit::down-button {
	subcontrol-origin:border;
	subcontrol-position:left;
	padding-right: 10px;
	image: url(:/cellculture_icons/arrow_left.png);
	width: 24px;height: 28px; left:5px;
}
QSpinBox::up-button:pressed, QDoubleSpinBox::up-button:pressed, QDateTimeEdit::up-button:pressed {
	subcontrol-origin:border;
	subcontrol-position:right;
	image: url(:/cellculture_icons/arrow_right.png);
	width: 23px;height: 27px;
}
QSpinBox::down-button:pressed, QDoubleSpinBox::down-button:pressed, QDateTimeEdit::down-button:pressed {
	subcontrol-position:left;
	image: url(:/cellculture_icons/arrow_left.png);
	width: 23px;height: 27px;
}
QSpinBox::disabled, QDoubleSpinBox::disabled, QDateTimeEdit::disabled {
	color:darkgray;
}

QComboBox {
	border: 1px solid white;
	padding: 1px 18px 1px 3px;
	min-width: 6em;
	font: 12pt &quot;微软雅黑&quot;;
	color:white;
}

QComboBox::hover{
	padding-top: 3px;
	padding-left: 4px;
}

QComboBox:editable {
	background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
	background: transparent;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
	background: transparent;
}

QComboBox:on {
	padding-top: 3px;padding-left: 4px;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: right;
	border: 0px solid white;
	width:40px;height:40px;
	border-top-right-radius: 3px;
	border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
	image: url(:/cellculture_icons/arrow_down.png);
	width: 24px;height: 28px;
}

QComboBox::down-arrow:on { 
	top: 1px;left: 1px;
}

QComboBox QAbstractItemView {
	color: white;
	border:0px solid white;
	selection-background-color: rgba(255, 255, 255, 50);
	background: blue;
}

QComboBox QAbstractItemView::item { 
	font: 12pt &quot;微软雅黑&quot;;
	min-height: 48px;
	min-width: 60px;
}
QGroupBox {
	font: 12pt &quot;微软雅黑&quot;;
}

QGroupBox::title {
	color: white;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="verticalSpacing">
      <number>30</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelDetectionMothed">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Detection Method:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxDetectionMothed">
       <property name="minimumSize">
        <size>
         <width>149</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSampleName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Sample Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditSampleName">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Sample</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelBwl">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Base Wave Length:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxBwl">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="suffix">
        <string> nm</string>
       </property>
       <property name="minimum">
        <number>190</number>
       </property>
       <property name="maximum">
        <number>850</number>
       </property>
       <property name="value">
        <number>340</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelChannelNumber">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Channel Number:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxChannelNumber">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelMeasureOptical">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Measure Optical:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxMeasureOptical">
       <property name="minimumSize">
        <size>
         <width>149</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelMethod">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Method:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxMethod">
       <property name="minimumSize">
        <size>
         <width>149</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelMwl">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Measure Wave Length:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QSpinBox" name="spinBoxMwl">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="suffix">
        <string> nm</string>
       </property>
       <property name="minimum">
        <number>190</number>
       </property>
       <property name="maximum">
        <number>850</number>
       </property>
       <property name="value">
        <number>260</number>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="labelCoefficient">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Coefficient:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxCoefficient">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelMixingLevel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Mixing Level:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="comboBoxMixingLevel">
       <property name="minimumSize">
        <size>
         <width>149</width>
         <height>48</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelMixingTime">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Mixing Time:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QSpinBox" name="spinBoxMixingTime">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="labelStaticTime">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string>Static Time:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QSpinBox" name="spinBoxStaticTime">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>48</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="suffix">
        <string> s</string>
       </property>
       <property name="maximum">
        <number>16777215</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>113</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
